tests/async/tasyncfile.nim

```diff
@@ -24,7 +25,7 @@ proc main() {.async.} =
     var file = openAsync(fn, fmAppend)
     await file.write("\ntest2")
     let errorTest = file.readAll()
-    echo await errorTest
+    yield errorTest
```

Comment:
The next line is ``doAssert errorTest.failed``, and running the test standalone results in an unhandled exception. Works both ways with ``koch test``.